### ISSUE TITLE
Support attribute selectors

### ISSUE THREAD

**Issue by jedmao** [postcss/postcss-bem-linter#72]
Any interest in supporting [attribute selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors) out of the box? LIke `.foo__button[disabled]`?

**Comment by davidtheclark** [comment]
It's done in SUIT (https://github.com/postcss/postcss-bem-linter/pull/35/files) --- but looks like you're using the BEM preset -- is that right? If so, yeah, we could alter the BEM regex just like we did the SUIT one.

**Comment by jedmao** [comment]
I'm using BEM, yes, but not the preset. Still, I figured those who are using the preset might benefit the same. I just added this to my custom component selectors:

```js
componentSelectors(block) {
    const element = `(?:__${WORD})?`;
    const modifier = `(?:--${WORD})?`;
    const attributes = `(?:\\[[^\\]]*\\])*`;
    return new RegExp(`^(?:\\.${block}${element}${modifier}${attributes})+$`);
}
```

**Comment by davidtheclark** [comment]
Released in 2.2.0.
